front/app/components/ReactionControl/ReactionButton.tsx

```diff
@@ -413,61 +416,86 @@ const ReactionButton = ({
         disabled={disabledReason === null}
         content={disabledMessage}
         trigger="mouseenter"
+        width="100%"
       >
-        <Button
-          buttonReactionMode={buttonReactionMode}
-          buttonReactionModeIsActive={buttonReactionModeIsActive}
-          reactingEnabled={buttonEnabled}
-          onMouseDown={removeFocusAfterMouseClick}
-          onClick={onClick}
-          className={[
-            className,
-            {
-              up: 'e2e-ideacard-like-button',
-              down: 'e2e-ideacard-dislike-button',
-            }[buttonReactionMode],
-            buttonReactionModeEnabled ? 'enabled' : '',
-          ].join(' ')}
-        >
-          <ReactionIconContainer
-            styleType={styleType}
-            size={size}
-            reactingEnabled={buttonEnabled}
-            buttonReactionModeIsActive={buttonReactionModeIsActive}
-            buttonReactionMode={buttonReactionMode}
-            disabledReason={disabledReason}
-          >
-            <ReactionIcon
-              name={iconName}
-              reactingEnabled={buttonEnabled}
-              buttonReactionModeIsActive={buttonReactionModeIsActive}
+        <>
+          {variant === 'text' && (
```

Comment:
This is a bit messy but it was the only way to easily plug the proposals button inside the idea reaction. I was thinking of duplicating / abstracting this logic and creating a separate component but it proved quite complicated. We should clean this up at some point but for now, this works.